stable/datadog/templates/daemonset.yaml

```diff
@@ -218,6 +218,16 @@ spec:
           - name: pointerdir
             mountPath: /opt/datadog-agent/run
           {{- end }}
+          {{- if (and (.Values.datadog.logsEnabled) (.Values.datadog.logsConfigContainerCollectAll)) }}
+          - name: logpodpath
+            mountPath: /var/log/pods
+            readOnly: true
+          {{- if .Values.datadog.containerLogsPath }}
+          - name: logcontainerpath
+            mountPath: {{ .Values.datadog.containerLogsPath | quote }}
```

Comment:
I'm wondering if the `mountPath` value should not depends on the variable, but instead be "static" like: `/var/log/containers`